2-mandatory/2-currency-code-lookup.js

```diff
@@ -18,7 +18,13 @@ const COUNTRY_CURRENCY_CODES = [
 ];
 
 function createLookup(countryCurrencyCodes) {
-  // write code here
+  let newCountryCurrencyCodes = Object.create(countryCurrencyCodes);
+  newCountryCurrencyCodes["GB"] = "GBP";
+  newCountryCurrencyCodes["DE"] = "EUR";
+  newCountryCurrencyCodes["NG"] = "NGN";
+  newCountryCurrencyCodes["MX"] = "MXN";
+
+  return newCountryCurrencyCodes;
```

Comment:
Although the outcome is current, the function is not correct, you define the object manually.

Reply:
function createLookup(countryCurrencyCodes) {
  // write code here
  let currencyObject ={};
for (let currency of countryCurrencyCodes) {
  currencyObject[currency[0]] = currency[1];
}  
  return currencyObject;  
}